packages/terra-visually-hidden-text/docs/README.md

```diff
@@ -0,0 +1,23 @@
+# Terra Visually Hidden Text
+
```

Comment:
Visually Hidden Text is ~text~ **a component** designed for screen readers that **renders text on the dom, but is not** ~should not be~ visible to the UI.  ->